Gym/FrozenLake.swift

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Python
+import TensorFlow
+
+let np = Python.import("numpy")
+let gym = Python.import("gym")
+
+_RuntimeConfig.printsDebugLog = false
+_RuntimeConfigP.printsDebugLog = false
+
+/// Solves the FrozenLake RL problem via Q-learning. This model does not use a
+/// neural net, and instead demonstrates Swift host-side numeric processing as
+/// well as python integration.
+
+let discountRate: Float = 0.9
+let learningRate: Float = 0.2
+let testEpisodeCount = 20
+
+typealias StateT = Int64
```

Comment:
```suggestion
typealias State = Int64
typealias Action = Int
```

Can `State` be changed to `Int` instead of `Int64`?

Reply:
done. yes `Int` works!